src/cmake/externalpackages.cmake

```diff
@@ -100,19 +100,59 @@ checked_find_package (pugixml 1.8 REQUIRED)
 
 
 # LLVM library setup
-checked_find_package (LLVM 7.0 REQUIRED
-                      PRINT LLVM_SYSTEM_LIBRARIES CLANG_LIBRARIES)
-# ensure include directory is added (in case of non-standard locations
-include_directories (BEFORE SYSTEM "${LLVM_INCLUDES}")
-link_directories ("${LLVM_LIB_DIR}")
+
+############ HACK ##############
+# On OSX, the Homebrew (and maybe any build) of LLVM 10.0 seems to have a
+# link conflict with its dependency on the llvm libc++ and the system
+# libc++, both can end up dynamically linked and lead to very subtle and
+# frustrating behavior failures (in particular, osl's use of libclang will
+# botch include file parsing any time LD_LIBRARY_PATH doesn't have the llvm
+# libc++ first).
+#
+# It seems that this is not a problem when linking against the llvm and
+# libclang libraries statically. So on apple and when LLVM 10+ are involved,
+# just force that choice. Other than larger executables, it seems harmless,
+# and in any case a better choice than this beastly bug.
```

Comment:
Out of curiosity, have you checked that this is still necessary in the new LLVM 11?

Reply:
I have not tested this on Mac (and just relied on the github CI). 
I could set up a build environment on my macbook but it will take some time (I don't even have brew installed :) ). Perhaps somebody else in the OSL community can help test this quicker ?